crypto/stack/stack.c

```diff
@@ -149,7 +149,7 @@ void *OPENSSL_sk_delete_ptr(OPENSSL_STACK *st, const void *p)
 
 void *OPENSSL_sk_delete(OPENSSL_STACK *st, int loc)
```

Comment:
I don’t think so.  Once it’s off the stack, the application can do what it wants.